03_Modules/Tenant/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@citrineos/tenant",
+  "version": "1.3.0",
+  "description": "The tenant module for OCPP v2.0.1. This module is not intended to be used directly, but rather as a dependency for other modules.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prepublish": "npx eslint",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "compile": "npm run clean && tsc -p tsconfig.json",
+    "clean": "rm -rf dist/* tsconfig.tsbuildinfo",
+    "fresh": "rm -rf node_modules package-lock.json && npm run clean"
+  },
+  "keywords": [
+    "ocpp",
+    "ocpp_v201"
+  ],
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/deasync-promise": "^1.0.0",
+    "eslint": "^8.48.0",
+    "typescript": "^5.0.4"
+  },
+  "dependencies": {
+    "@citrineos/base": "1.1.1",
```

Comment:
Probably should update these dependancies here.

Reply:
yup good point, updating this fixed the build also, woohoo!